### DISCUSSION TITLE
Add inputs support for Format Function `fmt`

### DISCUSSION THREAD

**Discussion by biscout42** [evidence-dev/evidence#2625] [Ideas]
### Description

According to the docs, [format function fmt](https://docs.evidence.dev/core-concepts/formatting/#format-function) accepts expression like

```
{fmt(expression, formatCode)}
```

I have tried to use `inputs` as expression and it didn't work. Example:

```
{fmt(inputs.my_input_name, 'usd0')}
```

outputs unformatted value from `inputs.my_input_name`. In this example `inputs.my_input_name` was a slider:

```
<Slider
    title="My slider name" 
    name=my_input_name
    size=large
    defaultValue=42
    min=0
    max=1000
    step=10
    fmt='usd0'
/>

```

### What problem would this solve?

That would allow to design pages with formatted values. One example: slider **current** value is not formatted (min/max are), even if add `fmt` property. One could output current slide value in the page text. 

### How should it work?

Format values from `inputs`.

**Comment by hughess** [comment]
A workaround is to wrap the value in `Number()` like so:

```js
{fmt(Number(inputs.my_input_name), 'usd0')}
```
